baiji/cli.py

```diff
@@ -39,6 +39,20 @@ def main(self, key):
             print e
             return 1
 
+class RestoreCommand(BaijiCommand):
+    DESCRIPTION = "restore deleted files on s3"
+    def main(self, prefix):
+        from baiji.util.console import LabeledSpinner
+        if s3.path.islocal(prefix):
+            raise ValueError("restore command only works on s3")
+        spin = LabeledSpinner()
+        for key in s3.ls(prefix, return_full_urls=True, require_s3_scheme=True, list_versions=True):
+            if not s3.exists(key):
+                spin.drop("Restoring deleted file {}".format(key))
+                s3.restore(key)
+            else:
+                spin.spin(key)
```

Comment:
Makes me want to try recovering files :)

[Edit: I just realize that I really only get the spinner if the files don't need recovering... so I can totally try this without actually recovering deleted files :D ]